tests/cross/src/test/scala/tests/pc/CompletionDocSuite.scala

```diff
@@ -152,7 +152,7 @@ class CompletionDocSuite extends BaseCompletionSuite {
        |""".stripMargin,
     compat = Map(
       "3" ->
-        """|sliding[B >: Int](size: Int, step: Int = 1): $1$.GroupedIterator[B]
+        """|sliding[B >: Int](size: Int, step: Int = 1): List[Int]#iterator.GroupedIterator[B]
```

Comment:
```suggestion
        """|sliding[B >: Int](size: Int, step: Int = 1): List[Int]#iterator.GroupedIterator[B]
```
any idea why this changed? Could you create a follow up issue to take a look at this?

Reply:
Maybe, I'll just add a comment to this task: https://github.com/scalameta/metals/issues/3915?

Reply:
Sure!